datreant/collections.py

```diff
@@ -321,17 +347,13 @@ def add_member(self, uuid, treanttype, basedir):
                 basedir of the new member in the filesystem
 
         """
-        if self.table.shape == (1, 0):
-            self.table = self._member2record(uuid, treanttype, basedir)
-        else:
-            # check if uuid already present
-            index = np.where(self.table['uuid'] == uuid)[0]
-            if index.size > 0:
-                # if present, update location
-                self.table[index[0]]['abspath'] = os.path.abspath(basedir)
-            else:
-                newmem = self._member2record(uuid, treanttype, basedir)
-                self.table = np.vstack((self.table, newmem))
+        # check if uuid already present
+        uuids = [member[0] for member in self.record]
+
+        if uuid not in uuids:
+            self.record.append([uuid,
```

Comment:
See above. Other branch that this one builds on now uses dicts for member records instead of lists. This still works since the member record is a list of dicts, maintaining ordering.